client/resource_group/controller/controller.go

```diff
@@ -221,6 +228,38 @@ func (c *ResourceGroupsController) Stop() error {
 	return nil
 }
 
+// Receive the consumption and flush it to the metrics.
+func (c *ResourceGroupsController) backgroundMetricsFlush(ctx context.Context) {
+	for {
+		select {
+		case <-ctx.Done():
+			return
+		case req := <-c.consumptionDispatcher:
+			consumption := req.GetConsumptionSinceLastRequest()
+			if consumption == nil {
+				continue
+			}
+			var (
+				name       = req.GetResourceGroupName()
+				rruMetrics = readRequestUnitCost.WithLabelValues(name)
+				wruMetrics = writeRequestUnitCost.WithLabelValues(name)
+			)
+			resourceGroupTokenRequestCounter.WithLabelValues(name).Inc()
+			// RU info.
+			if consumption.RRU != 0 {
+				rruMetrics.Observe(consumption.RRU)
```

Comment:
Does it's duplicated with the server side?

Reply:
@JmPotato @BornChanger Do u think it is necessary to count RU consumption of tidb instances?

Reply:
may do not need in current.

Reply:
I didn't really think about the need to view RU consumption at the instance level, so not keeping a record is okay for me.